src/tests/inductive-proofs.ts

```diff
@@ -0,0 +1,156 @@
+import {
```

Comment:
when is this run?

Reply:
never, but I hope to reconcile some Mina tests which are currently in snarkyjs/tests/integration and are written in raw JS w/o types, with this directory